src/main.js

```diff
@@ -99,6 +113,29 @@ function toggleIsFavorite(cardId) {
   renderIdeaCards(savedIdeaCards);
 }
 
+function addComment(cardId) {   //plus sign
+  ideaForm.classList.add('hidden');
+  commentForm.classList.remove('hidden');
+  commentCardId = cardId;
+}
+
+
+function createNewComment() {
+  event.preventDefault();
+  var newComment = new Comment(commentInputBox.value);
+  for (var i = 0; i < savedIdeaCards.length; i++) {
+    if (commentCardId === savedIdeaCards[i].id) {
+      savedIdeaCards[i].comments.push(newComment.data);
+    }
+  }
+  renderIdeaCards(savedIdeaCards);
+  //replace span with += commentInputBox.value
+  clearInputFields();
+}
+function displayIdeaForm() {
+  ideaForm.classList.remove('hidden');
```

Comment:
Should we make a `hide` and `show` function so we aren't repeating these lines throughout our code?  May make it a little more DRY